docs/how_to/single_node.md

```diff
@@ -16,9 +16,11 @@ directory on your host for durability:
 
 ```
 docker pull quay.io/m3/m3dbnode:latest
-docker run -p 7201:7201 -p 9003:9003 --name m3db -v $(pwd)/m3db_data:/var/lib/m3db quay.io/m3/m3dbnode:latest
+docker run -p 7201:7201 -p 7203:7203 -p 9003:9003 --name m3db -v $(pwd)/m3db_data:/var/lib/m3db -v $GOPATH/src/github.com/m3db/m3/src/dbnode/config/m3dbnode-local-etcd.yml:/etc/m3dbnode/m3dbnode.yml quay.io/m3/m3dbnode:latest
```

Comment:
Hm, you shouldn't have to add the config as a file mount - it will be auto added I believe, as per line 26 of the Dockerfile:
https://github.com/m3db/m3/blob/master/docker/m3dbnode/Dockerfile#L26